cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -84,7 +85,7 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
     }
 
     /**
-     * Translates a Python (https://docs.python.org/3/library/ast.html#ast.Assert] into a
+     * Translates a Python [Assert](https://docs.python.org/3/library/ast.html#ast.Assert) into a
```

Comment:
I think in the other functions we were using the syntax with backticks
```suggestion
     * Translates a Python [`Assert`](https://docs.python.org/3/library/ast.html#ast.Assert) into a
```